tox.ini

```diff
@@ -5,13 +5,7 @@ envlist = py26,py27,py33,py34
 # This is helpful to test installation but takes extra time
 skipsdist = True
 
-[testenv:py26]
-# Python 2.6 requires two extra test dependencies
-deps =
-    unittest2
-    -rrequirements.txt
-
 [testenv]
-deps = -rrequirements.txt
 commands =
-    nosetests {posargs:tests/unit}
+    {toxinidir}/scripts/ci/install
```

Comment:
It will but if you run it once, you will have all of the dependencies required for subsequent runs, meaning the install script runs pretty quickly (less than 5 seconds).